samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -816,17 +816,25 @@ Building with experimental support for Wi-Fi connectivity for nRF5340 DK with nR
 This sample :ref:`experimentally <software_maturity>` supports connecting to nRF Cloud using Wi-Fi instead of using LTE.
 
 An overlay for this is only provided for the nRF5340 DK with the nRF7002 EK shield attached.
+
 It is possible to use Wi-Fi with other hardware combinations (such as the nRF7002 DK), but you must adjust heap and stack usage accordingly.
 See the :file:`src/prj.conf` configuration file and the :file:`overlay_nrf7002ek_wifi_no_lte.conf` overlay for additional details.
 
-Connecting to nRF Cloud using Wi-Fi is currently only possible with MQTT, and only by using the experimental and insecure Kconfig option :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES`.
-See :ref:`nrf_cloud_multi_service_onboard_hardcoded` for details on setting it up.
+.. note::
+   Connecting to nRF Cloud using Wi-Fi is currently only possible with MQTT.
+
+.. important::
+   Connecting to nRF Cloud using Wi-Fi currently requires that device credentials be used insecurely.
+   The provided overlay for Wi-Fi connectivity uses the :ref:`TLS Credentials Subsystem <zephyr:sockets_tls_credentials_subsys>` (with the PSA Protected Storage backend, see :kconfig:option:`CONFIG_TLS_CREDENTIALS_BACKEND_PROTECTED_STORAGE`) to store credentials when not in use.
```

Comment:
@peknis This kconfig option, `CONFIG_TLS_CREDENTIALS_BACKEND_PROTECTED_STORAGE`, is from Zephyr and doesn't crosslink properly (or at least, it did not in my local builds, maybe it will work here)

Do you know how to link to a Zephyr KConfig?

Reply:
It works ok in the preview, just takes a bit time to pop up.